pkg/operator/clustermemberremovalcontroller/clustermemberremovalcontroller.go

```diff
@@ -132,6 +132,10 @@ func (c *clusterMemberRemovalController) attemptToScaleDown(ctx context.Context,
 		return fmt.Errorf("could not list etcd members: %w", err)
 	}
 
+	// removing etcd-boostrap from live voting members
+	// see https://issues.redhat.com/browse/OCPBUGS-10351
+	liveVotingMembers = removeBootstrapMember(liveVotingMembers)
+
 	if len(liveVotingMembers) < desiredControlPlaneReplicasCount {
 		klog.V(2).Infof("Ignoring scale-down since the number of etcd members (%d) < desired number of control-plane replicas (%d) ", len(liveVotingMembers), desiredControlPlaneReplicasCount)
```

Comment:
nit: Now that this might include bootstrap exclusion, did we want to soften "the number of etcd members" to  "the number of production etcd members" or "the number of non-bootstrap etcd members" or something?

Reply:
iiuc from haseeb comment, we want the number of voting member excluding the bootstrap member, to be more the desiredControlPlane 

cc @hasbro17

Reply:
> we want the number of voting member excluding the bootstrap member, to be more the desiredControlPlane

I agree with "non-bootstrap voters >= desired count" as the criteria.  I'm just quibbling with logs.  For example, say your desired count was 3 and you had one bootstrap and two non-bootstrap voters.  We want to ignore scale-down, because (2 non-bootstrap voters) < (3 desired count).  So far, so good.  But as your pull stands, we'll log:

> ... since the number of etcd members (2) < desired number of control-plane replicas (3)

But there are 3 voting members.  It's just that one of those members is the bootstrap node.  I want us to qualify  the logs to say something like:

> ... since the number of voting, non-bootstrap etcd members (2) < desired number of control-plane replicas (3)

And also, staring at this line for too long, I see the current code has `replicas (%d) "` with trailing whitespace, which should probably be `replicas (%d)"`.

Reply:
done